templates/cisco_ios_show_environment_temperature.template

```diff
@@ -0,0 +1,41 @@
+Value SWITCH (\d+)
+Value SWITCH_TEMPERATURE (\w+)
+Value INLET_TEMPERATURE_VALUE (\d+)
+Value INLET_TEMPERATURE_STATE (\w+)
+Value INLET_YELLOW_THRESHOLD (\d+)
+Value INLET_RED_THRESHOLD (\d+)
+Value HOTSPOT_TEMPERATURE_VALUE (\d+)
+Value HOTSPOT_TEMPERATURE_STATE (\w+)
+Value HOTSPOT_YELLOW_THRESHOLD (\d+)
+Value HOTSPOT_RED_THRESHOLD (\d+)
+
+Start
+  ^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$$
```

Comment:
I would add the following under L12
```
  ^Switch -> Continue.Record
```
And change L25 from `Record Start` to `Start`. This allows us to capture temperatures of other switches properly.